v21.1/partitioning.md

```diff
@@ -113,7 +113,13 @@ To ensure uniqueness or fast lookups, create a [secondary index](indexes.html) o
 
 Indexes are not required to be partitioned, but creating a non-partitioned index on a partitioned table may not perform well.
 
- When you create a non-partitioned index on a partitioned table, CockroachDB sends a [`NOTICE` message](https://www.postgresql.org/docs/current/plpgsql-errors-and-messages.html) to the client stating that creating a non-partitioned index on a partitioned table may not perform well.
+When you create a non-partitioned index on a partitioned table, CockroachDB sends a [`NOTICE` message](https://www.postgresql.org/docs/current/plpgsql-errors-and-messages.html) to the client stating that creating a non-partitioned index on a partitioned table may not perform well.
+
+#### Partition using foreign key reference
+
+If a partitioned table contains a [foreign key reference](foreign-key.html) to a non-partitioned table, the secondary index created auto-generated for the foreign key reference will not be partitioned. This can lead to performance impacts when querying against the partitioned table, as the data may exist in a distant node.
```

Comment:
```suggestion
If a partitioned table contains a [foreign key reference](foreign-key.html) to a non-partitioned table, the secondary index created automatically for the foreign key reference will not be partitioned. This can impact performance when querying against the partitioned table, as the data may exist in a distant node.
```